drivers/spi/Kconfig.sy1xx

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The device driver binding should be in the first commit (with the driver sources), then the second commit update the SoC's binding to include the spi node.

Reply:
alright, there we go.